docs/contributing/writing-docs-guide.md

```diff
@@ -235,6 +234,21 @@ Using {term}`React` makes frontends fun again!
 Using {term}`React` makes frontends fun again!
 
 
+#### Toggle paragraph (Exercises / FAQ)
+
```

Comment:
I like to see the actual code sample first, then show it rendered.

``````suggestion

To toggle showing or hiding content, use the `admonition` directive with the option of `:class: toggle`.

`````{code-block} rst
````{admonition} This is a title
:class: toggle

```{code-block} python
:linenos:
:emphasize-lines: 1, 3
a = 2
print("my 1st line")
print(f"my {a}nd line")
```
````
`````

``````